### PR TITLE
add test and stories on Modal

### LINKED ISSUE #506: Add tests and stories to `Modal` component
###Cover base test cases:
- modal appears on the screen with its content
- modal doesn't appear when `false` passed
- modal dissapear when clicking on `x` / out-of-modal area

###Write stories with `Storybook`

### PR DESCRIPTION
**Pull request by vyacheslav1618**
closes #506

test: close button check not finished
storybook: modal window not showing

### CODE REVIEW

**Comment by vyacheslav1618** [comment]
Modal hasn't a possibility to be written into StoryBook

whole of the tests are work

File: packages/frontapp/src/app/components/Modal/index.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+export default {
+  title: 'Modal',
+  component: Modal,
+};
+
+const Wrapper = (props: any) => {
+  const [active, setActive] = useState(true);
+  return <Modal active={active} setActive={setActive} {...props} />;
+};
```
Comment (Chichmarin17) [inline packages/frontapp/src/app/components/Modal/index.stories.tsx:13]: I can't see any content on Storybook when opening/closing a modal. Can you add some `children` to Modal e.g. a black square / text content ?

File: packages/frontapp/src/app/components/Modal/index.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+const Template: Story<IPropsModal> = (args: IPropsModal) => (
+  <Wrapper {...args} />
+);
+
+export const Primary = Template.bind({});
+Primary.args = {
+  active: true,
+  //setActive: ;
```
Comment (Chichmarin17) [inline packages/frontapp/src/app/components/Modal/index.stories.tsx:22]: Remove commented code

**Review by Chichmarin17 — CHANGES_REQUESTED** [review]

**Comment by vyacheslav1618** [comment]
There are some troubles with render Modal. Even though I try to set the "active" attribute to "true" or "false" it doesn't work. Modal component doesn't render. 
"Text for example #1" rendered. 
We need to refactor Modal completely and only then draw it up in the Storybook.

**Comment by Chichmarin17** [comment]
@vyacheslav1618 if you had checked how Modal works you would have noticed that it uses `React Portal`. That means, you have to add HTMLElement with `id` (`<div id='name_of_id'>`) that Modal uses to inject its content in. Also, change your `active` prop to use the Parent state

**Review by Chichmarin17 — APPROVED** [review]

**Pull request merged**